tests/when.test.ts

```diff
@@ -197,4 +197,12 @@ describe('when', () => {
         .with({ type: 'none' }, () => 'nope')
         .run();
   });
+
+
+  it('should pass matched value to otherwise', () => {
+    const result = match(42)
+      .with(51, d => d)
+      .otherwise(d => d);
+    expect(result).toBe(42);
+  })
```

Comment:
Since this is unrelated to the `match(...).when` method, I think it would be better to put this test in a separate `otherwise.test.ts` file